### ISSUE TITLE
Launch controller manager with joint state controller 

### ISSUE THREAD

**Issue by JavierRubia** [JavierRubia/Garrinator#2]
Launch controller manager with joint state controller

**Comment by andreucm** [comment]
Hi @JavierRubia 

In this branch: https://github.com/JavierRubia/Garrinator/tree/add_joint_state_controller
you can find the necessary code to execute the hardware interface and see how joints move at rviz. 
Please test it with the hardware following the steps: 
1. git fetch
1. git checkout add_joint_state_controller
1. roscd && cd ..
1. catkin_make
1. roslaunch garrinator_hardware_interface hw_if.launch

In case of success you should merge the branch to master. If no success, please let me know the error message, and/or try to fix it.

thanks

andreu

**Comment by JavierRubia** [comment]
Hi @andreucm 

I've been testing it but it has an **error in rviz**  all the rims and the rollers are in the origin frame due an error: `No transform from [any_link] to [base_link]`. I've been trying to figure out what's the problem.

I've find out is the controller have to be updated so I've added it in : https://github.com/JavierRubia/Garrinator/blob/add_joint_state_controller/hardware_interface/src/test/hw_if_test.cpp#L30 thinking that maybe the information was not flowing trough the controller but that's not the case.

Also if https://github.com/JavierRubia/Garrinator/blob/add_joint_state_controller/description/launch/display.launch#L9 gets uncommented the wheels appear to be in the right place but it wont move anyway.
 
Long story short: the wheels are not in the right place and either move.
